tests/e2e/volumepropverification_test.go

```diff
@@ -24,118 +24,122 @@ import (
 	"log"
 	"os"
 	"strings"
-	"testing"
-	"time"
 
+	con "github.com/vmware/docker-volume-vsphere/tests/constants/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/verification"
+	. "gopkg.in/check.v1"
 )
 
-var (
-	esx              = os.Getenv("ESX")
-	volSizes         = []string{"100MB"}
-	vms              = []string{os.Getenv("VM1")}
-	formatTypes      = []string{"thin", "zeroedthick", "eagerzeroedthick"}
-	vmIP             string
-	dockerVolmRmvCmd = "docker volume rm "
-	dockerCliCheck   bool
-	containerName    string
-)
+type VolumePropVerificationSuite struct {
+	volumeName     string
+	dockerHostIP   string
+	dockerHostName string
+	containerName  string
+	esxIP          string
+}
+
+func (s *VolumePropVerificationSuite) SetUpTest(c *C) {
+	s.dockerHostIP = inputparams.GetDockerHostIP("VM2")
+	s.dockerHostName = govc.RetrieveVMNameFromIP(s.dockerHostIP)
+	s.esxIP = os.Getenv("ESX")
+	s.volumeName = inputparams.GetVolumeNameWithTimeStamp("volumepropverification_test")
+	s.containerName = inputparams.GetContainerNameWithTimeStamp("volumepropverificationSuite_test")
```

Comment:
Can check if any of the needed params is not valid and call c.Skip() so that the test is skipped vs. running and spewing errors or Assert()s.